### PR TITLE
Update TR Geothermal Values

### PR DESCRIPTION
**Pull request by SSD1805**
Added Geothermal CO2 energy output as per issue 5019

## Issue
https://github.com/electricitymaps/electricitymaps-contrib/issues/5019#issue-1569478266
## Description
 Verify that the values in the paper are valid.
 Create a PR to update the regional emissions if the above task is done.

### Preview

image

### CODE REVIEW

File: config/zones/TR.yaml
Comment (VIKTORVAV99) [inline config/zones/TR.yaml:130]: This should follow MLA citation style as and should not contain a line break (`\n`). If needed, you can wrap the whole string in single quotes (`''`).

File: config/zones/TR.yaml
Comment (VIKTORVAV99) [inline config/zones/TR.yaml:128]: The datetime listed in the paper for the value you added is August 2019, not January 2020.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
First, I'm not sure it's as easy as this PR makes it out to be. The paper in question uses different values per year so we should as well, preferably going back as far as 2015 till today (or 2024 as the paper suggest)

Secondly, we need to verify these values to make sure they can be trusted.

Thirdly, this PR does not follow the structure that is required for the zone files, which will need to be corrected before this can be merged.

File: config/zones/TR.yaml
Comment (mathilde-daugy) [inline config/zones/TR.yaml:131]: url is not a valid key for this config, we only use source to link to the research paper

Comment (mathilde-daugy) [reply config/zones/TR.yaml:131]: In [KR.yaml](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/config/zones/KR.yaml), there is a url linked for unknown using the key `_url`

File: config/zones/TR.yaml
Comment (VIKTORVAV99) [inline config/zones/TR.yaml:134]: Why is this listed under unknown?

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
There are still highly questionable changes in here and this will fail in our backend.

Please do take a look how the previous regional sources are formatted.

We also need to make sure we get a accurate value for each year the paper cover before this can be merged, right now we only have data for the end of 2019. We also need to verify these numbers as we can't just take it at face value.

**Comment by VIKTORVAV99** [comment]
@SSD1805 have you made any progress on this incorporating the feedback here? We can't merge this as is.

**Comment by VIKTORVAV99** [comment]
Closing this PR as there has been no response from the author.

**Pull request closed without merging**
